pkgs/development/ocaml-modules/crowbar/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildDunePackage, fetchFromGitHub, ocplib-endian, cmdliner, afl-persistent
+, calendar, fpath, pprint, uutf, uunf, uucp }:
+
+buildDunePackage rec {
+  pname = "crowbar";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner  = "stedolan";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "0wjfc9irvirfkic32ivvj6qb7r838w08b0d3vmngigbjpjyc9b14";
+  };
+
+  minimumOCamlVersion = "4.08";
+
+  # disable xmldiff tests, so we don't need to package unmaintained and legacy pkgs
+  postPatch = "rm -rf examples/xmldiff";
+
+  propagatedBuildInputs = [ ocplib-endian cmdliner afl-persistent ];
+  checkInputs = [ calendar fpath pprint uutf uunf uucp ];
+  doCheck = true;
```

Comment:
Would `crowbar` build on Aarch-64 if tests were disabled on that platform?

Reply:
Yes, it seems so. Disabled tests on Aarch64 and crowbar as well as index build fine on ofborg now (can't test it myself unfortunately).